Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -4799,5 +4799,24 @@ RefPtr<CSSPrimitiveValue> consumeAnchor(CSSParserTokenRange& range, CSSParserMod
     return CSSPrimitiveValue::create(anchor);
 }
 
+RefPtr<CSSValue> consumeViewTransitionTypes(CSSParserTokenRange& range)
+{
+    if (range.peek().id() == CSSValueNone)
+        return consumeIdent(range);
+
+    CSSValueListBuilder list;
+    do {
+        if (range.peek().id() == CSSValueNone)
+            return nullptr;
+        auto type = consumeCustomIdent(range);
+        if (!type)
+            return nullptr;
+        if (type->customIdent().startsWith("-ua"_s))
```

Comment:
The correct prefix is `-ua-` (reference: https://github.com/w3c/csswg-drafts/issues/9534#issuecomment-1802364085)

Anyway I think we should do this `-ua-` filtering in one place, namely within `ViewTransitionTypeSet`. This avoids having two identical filtering code (one here, one when matching `:active-view-transition-type()` selector). What do you think?

Reply:
That seems reasonable.

I'll have to create and store a `ViewTransitionTypeSet` inside `StyleRuleViewTransition`, and I can leave this subtest failing until you merge your PR.